deployment/ansible/group_vars/all

```diff
@@ -4,19 +4,19 @@ developing_or_testing: "'development' in group_names or 'test' in group_names"
 not_developing: "'development' not in group_names"
 not_testing: "'test' not in group_names"
 
-java_version: "7u71-2.5.3-0ubuntu0.12.04.1"
+java_version: "7*"
 
-python_version: "2.7.3-0ubuntu2.2"
-pip_version: "1.0-1build1"
+python_version: "2.7.*"
+pip_version: "1.*"
 
-postgis_liblwgeom_package_version: "2.1.3+dfsg-1.pgdg12.4+7"
-postgis_package_version: "2.1.5+dfsg-1~88.git69d0152.pgdg12.4+1"
+postgis_liblwgeom_package_version: "2.1.*"
+postgis_package_version: "2.1.*"
 
-postgresql_package_version: "9.3.5-2.pgdg12.4+1"
+postgresql_package_version: "9.3.*"
```

Comment:
I've been getting this to work reliably on Trusty:

```yaml
postgresql_package_version: "9.3.5*.pgdg14.04+1"
```

Unfortunately, I can't get more than one blob to work. I'm trying to pin at 9.3.5 because that's the version in the NYC CloudFormation.